Gemfile

```diff
@@ -46,9 +51,8 @@ gem 'truncato'
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'factory_bot_rails'
+  gem 'rspec_junit_formatter'
   gem 'rspec-rails'
-  gem 'rubocop', '~> 0.85.1', require: false
-  gem 'sqlite3'
 end
```

Comment:
All of the gems you added or moved should remain in the `group :development, :test do` section.  That includes sqlite, coveralls, rails-controller-testing, rubocop, and simplecov.